geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateRegionCommand.java

```diff
@@ -252,7 +253,7 @@ public Result createRegion(
               CliStrings.CREATE_REGION__MSG__OPTION_0_CAN_BE_USED_ONLY_FOR_PARTITIONEDREGION,
               regionFunctionArgs.getPartitionArgs().getUserSpecifiedPartitionAttributes()) + " "
               + CliStrings.format(CliStrings.CREATE_REGION__MSG__0_IS_NOT_A_PARITIONEDREGION,
-                  useAttributesFrom));
+                  regionPath));
```

Comment:
Can you explain the reason behind this change? I couldn't quite follow it.

Reply:
I think that this was previously a cut-n-paste from above. Previously the error string would show the region as `"null"` instead of the correct name.